itou/employee_record/models.py

```diff
@@ -225,8 +225,8 @@ class Meta(ASPExchangeInformation.Meta):
 
     def __str__(self):
         return (
-            f"PK:{self.pk} PASS:{self.approval_number} SIRET:{self.siret} JA:{self.job_application} "
-            f"JOBSEEKER:{self.job_application.job_seeker} STATUS:{self.status}"
+            f"PK:{self.pk} PASS:{self.approval_number} SIRET:{self.siret} JA:{self.job_application.pk} "
```

Comment:
Évitera peut-être un jour une requête.
```suggestion
            f"PK:{self.pk} PASS:{self.approval_number} SIRET:{self.siret} JA:{self.job_application_id} "
```